openshift/templates/bcfw_p1_forecasts.cronjob.yaml

```diff
@@ -0,0 +1,73 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: ${JOB_NAME}-cronjob-template
+  annotations:
+    description: "Scheduled task to download noon-time weather forecasts for all weather stations from the BC Fire Weather Phase 1 API."
+    tags: "cronjob,bcfw-p1"
+labels:
+  app.kubernetes.io/part-of: "${NAME}"
+  app: ${NAME}
+parameters:
+  - name: NAME
+    description: Module name
+    value: wps-api
+  - name: SUFFIX
+    description: Deployment suffix, e.g. pr-###
+    required: true
+  - name: PROJECT_TOOLS
+    value: auzhsi-tools
+  - name: JOB_NAME
+    value: bcfw-p1-forecasts
+objects:
+  - kind: CronJob
+    apiVersion: batch/v1beta1
+    metadata:
+      name: ${JOB_NAME}
+      labels:
+        cronjob: ${JOB_NAME}
+    spec:
+      schedule: "30 8,16 * * *"
+      concurrencyPolicy: "Forbid"
```

Comment:
Just an f.y.i. - keep and eye on your cronjob. The env Canada one got stuck, and never stopped, so other jobs never started up. Which is good and bad - just monitor this job for a while to make sure it doesn't have similar issues.

Reply:
Interesting. Maybe we can go with `"Replace"` instead?

Reply:
I like that - yeah - let's go with Replace here instead of Forbid!

Reply:
```suggestion
      concurrencyPolicy: "Replace"
```